chef/cookbooks/provisioner/templates/default/autoyast-upgrade.xml.erb

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <deploy_image>
+    <image_installation config:type="boolean">false</image_installation>
+  </deploy_image>
+  <general>
+    <ask-list config:type="list"/>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+      <final_halt config:type="boolean">false</final_halt>
+      <final_reboot config:type="boolean">false</final_reboot>
+      <halt config:type="boolean">false</halt>
+      <second_stage config:type="boolean">true</second_stage>
+    </mode>
+    <mouse>
+      <id>none</id>
+    </mouse>
+    <proposals config:type="list"/>
+    <signature-handling>
+      <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+      <accept_verification_failed config:type="boolean">true</accept_verification_failed>
+      <import_gpg_key config:type="boolean">true</import_gpg_key>
+    </signature-handling>
+    <storage/>
+  </general>
+  <add-on>
+    <add_on_products config:type="list">
+    <% @repos.keys.sort.each do |name| %>
+      <listentry>
+        <name><%= name %></name>
+        <product><%= name %></product>
+        <product_dir>/</product_dir>
+        <media_url><%= @repos[name][:url] %></media_url>
+        <!-- See bnc#787157 for discussion on ask_on_error -->
+        <ask_on_error config:type="boolean"><%= @repos[name][:ask_on_error] || 'false' %></ask_on_error>
+      </listentry>
+    <% end %>
+    </add_on_products>
+  </add-on>
+  <timezone>
+    <hwclock>UTC</hwclock>
+    <timezone><%=@timezone%></timezone>
+  </timezone>
+  <services-manager>
+    <services>
+      <enable config:type="list">
+        <service>sshd</service>
+      </enable>
+    </services>
+  </services-manager>
+  <software>
+    <packages config:type="list">
+      <package>autoyast2-installation</package>
+<% if @architecture == "x86_64" -%>
+      <package>biosdevname</package>
+<% end -%>
+      <package>netcat</package>
+      <package>ruby2.1-rubygem-chef</package>
+<% if @is_ses -%>
+      <package>ses-release</package>
+<% else -%>
+      <package>suse-openstack-cloud-release</package>
+      <package>supportutils-plugin-suse-openstack-cloud</package>
+<% end -%>
+<% @packages.each do |package| -%>
+      <package><%= package %></package>
+<% end -%>
+    </packages>
+    <patterns config:type="list">
+      <pattern>Minimal</pattern>
+      <pattern>base</pattern>
+    </patterns>
+    <!-- Avoid default Cloud pattern "cloud_admin" to be installed on all nodes -->
+    <remove-patterns config:type="list">
+	<pattern>cloud_admin</pattern>
+    </remove-patterns>
```

Comment:
IIRC YaST  was selecting the default pattern for add-on when upgrading. But it seems we removed the default pattern options from the image meanwhile so I guess this can go away now.